coalib/bearlib/abstractions/Linter.py

```diff
@@ -368,6 +369,43 @@ def process_output_corrected(self,
                              diffs={filename: diff},
                              severity=diff_severity)
 
+        def process_output_unidiff(self,
```

Comment:
I would call the function `process_output_unified_diff`, I think that makes it easier for the user to find that function when he writes a custom `process_output` function^^